### DISCUSSION TITLE
Cancel annual subscription

### DISCUSSION THREAD

**Discussion by xeon123** [community/community#122713] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have enrolled to the copilot annual subscription. Can I cancel the subscription while it is available, and will I get back the remaining payment?

**Comment by hafiz-muhammad** [comment, marked as answer]
[Canceling your Copilot Individual subscription](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-individual-subscription#canceling-your-copilot-individual-subscription)

This is directly from GitHub Docs:

_You can cancel your Copilot Individual subscription at any time. The cancellation will take effect at the end of your current billing cycle. You can also cancel your Copilot Individual trial, during the 30-day trial period. For more information, see "[Canceling your Copilot Individual trial](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-individual-subscription#canceling-your-copilot-individual-trial)."_

**Reply by EConway1** [reply]
Refund my credit card.

**Reply by EConway1** [reply]
Please refund my Capital One credit card.

**Reply by EConway1** [reply]
Full $100 refund is owed. On May 31st told you I did NOT want Copilot.

**Reply by hafiz-muhammad** [reply]
@EConway1 It would be better for you to reach out to GitHub support. I don't work for GitHub.

**Reply by Meredock** [reply]
lol

**Reply by Meredock** [reply]
he told you he didn't want it

**Reply by gtg422b** [reply]
Trying to do the same thing - fucking criminals.  They make it so you can't cancel or hide it refuse to give support to find it etc ... .

**Reply by logik360** [reply]
rediculous ...I am tryign to find since 45 minutes where I can cancel my subscription ...

**Reply by Meredock** [reply]
Theres literally nothing in this link about cancelling github copilot..

**Reply by Meredock** [reply]
This is the correct page https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/managing-your-copilot-plan/viewing-and-changing-your-copilot-plan

**Reply by Cactusinhand** [reply]
The Github Docs sucks, there is no way to cancel Copolit subscription from the link your post, it is a completly wrong link.

**Reply by m-coder567** [reply]
github is no updates and some that bugs oh man shits

**Comment by chevfit29** [comment]
Really cancel my subscription. I really don’t want it.

**Reply by m-coder567** [reply]
totaliy you loss not get paid so pleas cancel my suscription

**Comment by RT-60** [comment]
CANCEL MY SUBSCRIPTION WHICH IS "COMMERCIAL VEHICLE INSURANCE & COMPLIANCE SERVICE ... THANK YOU GARY BURLEY

**Comment by RT-60** [comment]
CANCEL MY SUBSCRIPTION WHICH IS "COMMERCIAL VEHICLE INSURANCE & COMPLIANCE SERVICE ... THANK YOU GARY BURLEY

**Comment by RT-60** [comment]
CANCEL MY SUBSCRIPTION WHICH IS "COMMERCIAL VEHICLE INSURANCE & COMPLIANCE SERVICE ... THANK YOU GARY BURLEY

**Comment by sjboswell** [comment]
Cancel my subscription

**Reply by davidrufael21gmailcom** [reply]
Cancel my subscription pls

**Comment by lamlong** [comment]
i need cancel my subciption github can you help me many thanks

**Comment by lamlong** [comment]
hi ;  i need cancel mysubcription to github can you help me thanks

**Comment by Steed-13Oct** [comment]
I canceled my subscription last year, and I am still getting charges in my account. I need to talk to customer service.

**Reply by Jindequan** [reply]
Did you finish the conversation?

**Comment by Meredock** [comment]
Where is my mother

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Jindequan** [comment]
I have already cancelled my copilot subscription. But it still cost my 10 from visa few hours ago. Why? How to rollback?

**Comment by damodharm10** [comment]
year i also want to cancel, i did 1st monthly and then accidently yearly, need to cancel yearly and roll back funds please

**Comment by longvireak** [comment]
i need to cancel my subscription too

**Comment by damodharm10** [comment]
subscription cancel no return .

**Reply by damodharm10** [reply]
my monthly subscription already closed yearly was canceled already no action from github

**Comment by IBrenoS** [comment]
Para um fórum comunitário, não achar a solução é frustrante.

**Comment by NguyeTin** [comment]
Dear GitHub Support Team,

I am writing to request a refund for an accidental purchase of GitHub Pro.

I intended to subscribe to GitHub Copilot, but mistakenly purchased the GitHub Pro plan on [March 1, 2026]. I realized my mistake shortly after the charge.

I have already downgraded my subscription from GitHub Pro to the Free plan to prevent further charges.

Could you please review my account and process a refund for the GitHub Pro subscription fee? My GitHub username is NguyenTin

Thank you for your time and assistance.

Sincerely,
NguyenTin
